client/team_project_assignment_test.go

```diff
@@ -0,0 +1,122 @@
+package client_test
+
+import (
+	"errors"
+	. "github.com/env0/terraform-provider-env0/client"
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+const dummyProjectAssignmentId = "dummyId"
+const dummyProjectId = "dummyProjectId"
+const dummyProjectRole = "Admin"
+const dummyTeamId = "dummyTeamId"
+
+var _ = Describe("TeamProjectAssignment", func() {
```

Comment:
Where are the tests for your argument's validations? like missing field you have checked

Reply:
https://github.com/env0/terraform-provider-env0/blob/f1731c2be50a438b8c6fd40c4c093181726081ec/client/team_project_assignment_test.go#L52

Reply:
>ForceNew indicates that any change in this field requires the resource to be destroyed and recreated.

https://www.terraform.io/docs/extend/schemas/schema-behaviors.html#forcenew